health/notifications/alarm-notify.sh.in

```diff
@@ -2208,6 +2275,7 @@ for state in "${SENT_EMAIL}" \
 	"${SENT_CUSTOM}" \
 	"${SENT_IRC}" \
 	"${SENT_AWSSNS}" \
+    "${SENT_MATRIX}" \
```

Comment:
Indenting is off here, please make sure it matches up with the surrounding code.

Reply:
Pushed a commit to resolve this. My editor was using the wrong whitespace characters